package.json

```diff
@@ -16,6 +16,7 @@
     "url": "git://github.com/senecajs/seneca-refer.git"
   },
   "scripts": {
+    "format": "prettier --write .",
```

Comment:
rename this to `prettier` as this is the convention used in other seneca plugins

Reply:
Understood. Thanks for highlight this. Changes made on this commit: https://github.com/senecajs/seneca-refer/pull/6/commits/02a6a70a2483d1ece7d90527422f22fb5256d6ab#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R19